ui/app/utils/build-api-validators.js

```diff
@@ -0,0 +1,26 @@
+import { validator, buildValidations } from 'ember-cp-validations';
+
+/**
+ * Add validation on dynamic form fields generated via open api spec
+ * For fields grouped under default category, add the require/presence validator
+ * @param {Array} fieldGroups
```

Comment:
Can we get a little more description here about what shape the objects in the array should be in? Or where to find it?

Reply:
Good point. Added example for fieldgroup structure.